blender/arm/logicnode/object/LN_get_visible.py

```diff
@@ -1,12 +1,15 @@
 from arm.logicnode.arm_nodes import *
 
 class GetVisibleNode(ArmLogicTreeNode):
-    """Get visible node"""
+    """Get Visible node"""
```

Comment:
To me it looks like this line is responsible for the conflicts, the GetVisible node docstring in current master doesn't end with "node" but here it is removed. You can try to locally merge the master branch into your branch to resolve the conflicts.